recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+    --disable-network \
+    --disable-static
```

Comment:
Yeah, this disables static libraries, we would like them enabled in case users want them.